fl4health/clients/basic_client.py

```diff
@@ -270,8 +295,23 @@ def evaluate(self, parameters: NDArrays, config: Config) -> Tuple[float, int, Di
         if not self.initialized:
             self.setup_client(config)
 
+        _, _, current_server_round = self.process_config(config)
```

Comment:
If you just need to extract the `current_server_round` here you could just use
```
current_server_round = self.narrow_config_type(config, "current_server_round", int)
```
As it's slightly more pointed.